packages/type-utils/src/isTypeReadonly.ts

```diff
@@ -15,6 +20,7 @@ const enum Readonlyness {
 
 export interface ReadonlynessOptions {
   readonly treatMethodsAsReadonly?: boolean;
+  readonly allow?: Array<TypeOrValueSpecifier>;
```

Comment:
we used to have the `array-type` rule turned on.
When we removed it from the recommended set we forgot to turn it on in our repo!

```suggestion
  readonly allow?: TypeOrValueSpecifier[];
```

Reply:
update: enabled this on the v6 branch

Reply:
Thanks, fixed :)